merlin/loader/ops/embeddings/tf_embedding_op.py

```diff
@@ -0,0 +1,245 @@
+import numpy as np
+import tensorflow as tf
+
+from merlin.core.protocols import Transformable
+from merlin.dag import BaseOperator
+from merlin.dag.selector import ColumnSelector
+from merlin.schema import ColumnSchema, Schema, Tags
+
+
+class TFEmbeddingOperator(BaseOperator):
```

Comment:
Most of these are repeated with small tweaks, would be nice to be able to converge so we dont have three operators for the same thing just using different inputs.

Reply:
Seems like that's the best we can currently do though, so 🤷🏻

Reply:
Would your proposal for columns remove the need for ColumnSchema?

I'm guessing Tags is significantly different but I could be wrong.